### ISSUE TITLE
Consider logging emails instead of sending them

### ISSUE THREAD

**Issue by chriscct7** [norcross/airplane-mode#43]
In airplane mode, emails shouldn't really send out. I suggest logging them instead of sending them out, including or maybe doing something like https://github.com/curtismchale/WPTT-Basic-Email-Logging

**Comment by johnbillion** [comment]
Personally I'd have no interest in logging emails, but it depends what you use Airplane Mode for. Preventing them from attempting to send makes sense though. It'd just need something like this:

```
if ( ! function_exists( 'wp_mail' ) ) {
	function wp_mail() {
		return true;
	}
}
```

**Comment by chriscct7** [comment]
Let me rephrase the feature idea. When on, block emails out. And with a filter maybe show a table of all attempted outbound connections whether mail, remote post or get, etc

**Comment by norcross** [comment]
the blocking has been added. @chriscct7 if you can see a place to add a hook / filter to log said email, I'm all for it, but I think you can do that outside of the plugin itself

**Comment by chriscct7** [comment]
Yeah I agree. Abandoning that idea
